BEIMA.Client/src/pages/Devices/DevicePage.js

```diff
@@ -110,6 +118,29 @@ const DevicePage = () => {
     )
   }
 
+  /**
+   * Renders a card with a dropdown field input. 
+   * 
+   * @param editable: can this input be used
+   * @param id: id that should be set on the input
+   * @param label: label of the input
```

Comment:
missing dropDownText, items, and buttonStyle

Reply:
fixed 5bad63f